src/components/hooks/use-auth.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react'
+import { isLoggedIn } from 'common/utility'
+
+const useAuth = () => {
```

Comment:
```suggestion
const useAuth = () => {
```
`useAuth` sounds misleading 🤔 we should name it something understandable like `useAuthCheck`